azure-pipelines.yml

```diff
@@ -26,6 +26,10 @@ steps:
 
 - script: |
     pip install pytest pytest-azurepipelines
-    pytest fitbenchmarking\utils\tests\test_logger.py
-#    pytest fitbenchmarking\utils\tests\test_fitbm_result.py (breaks in first azure pipeline install)
+    pytest fitbenchmarking\utils
```

Comment:
I think it would be nice to make an `azure` folder similar to the `travis` folder and have two scripts in it, one to install (this would be super short) and the other to run the tests

Reply:
will address this in separately created issue

Reply:
Is there a particular reason for this? It is definitely in scope for this issue and will make sure that any future tests are put in correctly

Reply:
Yes. I could hack something together, but in my opinion it deserve some thinking, is not urgent, and it would be recommended to check a few other repos and advice on this before splitting